src/mlpack/core/util/version.hpp

```diff
@@ -17,7 +17,7 @@
 // The version of mlpack.  If this is a git repository, this will be a version
 // with higher number than the most recent release.
 #define MLPACK_VERSION_MAJOR 4
-#define MLPACK_VERSION_MINOR 1
+#define MLPACK_VERSION_MINOR 2
 #define MLPACK_VERSION_PATCH 1
```

Comment:
@rcurtin since this is for mlpack 4.2.0, should that be `#define MLPACK_VERSION_PATCH 0` rather than `1`?